src/3p-frame.js

```diff
@@ -229,9 +229,6 @@ export function getDefaultBootstrapBaseUrl(parentWindow, opt_srcFileBasename) {
 }
 
 function getAdsLocalhost(win) {
-  if (urls.localDev) {
-    return `//${urls.thirdPartyFrameHost}`;
-  }
   return 'http://ads.localhost:'
```

Comment:
this might not work on heroku though

Reply:
Is there a separate check for heroku? We can replace `if (urls.localDev)` with that check.

Reply:
@lannka Can you confirm that this doesn't work for heroku? If so, is there a valid check, given that this is runtime code and not infra code?

Reply:
I'm not aware of any. Maybe just check the existence of `urls.thirdPartyFrameHost` (which is from AMP_TESING_HOST env var)?

Reply:
> Can you confirm that this doesn't work for heroku?

See this: http://amphtml-nightly.herokuapp.com/examples/ads.amp.html

Reply:
I see. The heroku flow, and more generally, 3P ads on a non-localhost server were already broken prior to this PR. @bradfrizzell is working separately on fixing the issue. See Item 2 in https://github.com/ampproject/amphtml/issues/12054#issuecomment-346154439.

This PR should not affect that flow, so I'm going to merge it. The non-localhost issue can be addressed separately.